Makefile

```
@@ -115,11 +110,15 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet setup-envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
+.PHONE: prepare-release
+prepare-release:
+	./hack/prepare-release.sh
+
 ##@ Build
 
 .PHONY: build
 build: generate fmt vet ## Build manager binary.
-	go build -o bin/manager main.go
+	go build -o bin/manager -ldflags ${LD_FLAGS} main.go
```

Wait, i assume we will need:
```suggestion
	CGO_ENABLED=0 go build -o bin/manager -ldflags ${LD_FLAGS} main.go
```